docs/ref/forms/api.txt

```diff
@@ -932,6 +932,50 @@ and using the template above, would render something like:
 
     <label for="myFIELD">...</label><input id="myFIELD" type="text" name="my_field" />
 
+.. _customizing-bound-field:
+
+Customizing ``BoundField``
+--------------------------
+
+.. versionadded:: 1.9
+
+If you are in a situation where using a custom subclass of
+:class:`~django.forms.Field` isn't enough, consider customizing
+:class:`~django.forms.BoundField` additionally.
+
+A subclass of :class:`~django.forms.Field` can override ``bind_to_form()``.
+This method takes an instance of :class:`~django.forms.Form` and the name of
+the field. Its return value will be used when accessing the field in a
+template. Most likely it will be an instance of (a subclass) of
+:class:`~django.forms.BoundField`.
+
+If you have a ``GPSCoordinatesField`` for example and want to be able to access
+additional information about the coordinates in a template, this could be
+implemented like follows::
+
+    class GPSCoordinatesBoundField(BoundField):
+        @property
+        def country(self):
+            """
+            Returns the country the coordinates lie in or None if none can be
+            determined.
+            """
+            if self.value():
```

Comment:
intermediate variable might be appropriate to avoid calling value() twice?
```
value = self.get_value()
return get_country_from_coordinates(value) if value else None
```